invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/UserDetails/components/ItemsSearch/__tests__/ItemsSearch.js

```diff
@@ -51,9 +51,11 @@ describe('PatronLoans tests', () => {
     const mockedClear = jest.fn();
     const mockedCheckoutItem = jest.fn();
 
-    const items = [
-      { item_pid: '4', barcode: '3', document_pid: 4, status: 'LOANABLE' },
-    ];
+    const items = {
+      hits: [
+        { item_pid: '4', barcode: '3', document_pid: 4, status: 'LOANABLE' },
```

Comment:
document_pid also a string, we are struggling a lot with the mix of integers and strings for the pids, we will have to address this on a project level.

Reply:
everywhere it is string except for user where the API returns a number, and maybe we should just serialize that when we get the response. The problem is coming from the fact that the user is not a record.